### DISCUSSION TITLE
Raw string coloring

### DISCUSSION THREAD

**Discussion by BMBurstein** [microsoft/vscode-cpptools#8715] [Q&A] [answered]
I just discovered a cool feature by accident: If I wrap my raw string containing sql in `R"sql(...)sql"` I get sql coloring in the string. This is (in my opinion) a really cool feature, but it seems to only work for sql.

1. Is this feature documented anywhere? I ran into it by accident
2. Can this be extended to other languages?

See here an example of what I am talking about:
![image](https://user-images.githubusercontent.com/1149432/150948069-18f5d1a2-51b7-48dd-9116-efb5a784976e.png)

**Comment by sean-mcmanus** [comment, marked as answer]
It's implemented at https://github.com/jeff-hykin/better-cpp-syntax/blob/db3f4e4a5d8335b2f6d689bec490c23f8313630f/main/patterns/raw_strings.rb which is used by VS Code and not our C/C++ extension.
